src/theme/App/Layout.tsx

```diff
@@ -0,0 +1,20 @@
+import React, {ReactChildren, ReactChild} from 'react';
+
+import Header from 'src/components/Header';
+
+interface AppProps {
+    children: ReactChild | ReactChildren;
+}
+
+const Layout = ({children}: AppProps) => {
```

Comment:
So if you do something like this you don't need to explicitly pass `children` as a prop type
```suggestion
const Layout: React.FC = ({ children }) => {
```

And in case you have some props 
```suggestion
interface AppProps {
    firstName: string // dummy
}

const Layout: React.FC<AppProps> = ({ children, firstName }) => {
```

Reply:
Thanks for explanation, I have made changes according your suggestion.